.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,6 +23,3 @@ Fixes #
 ```release-note
 
 ```
-
```

Comment:
should we replace this with the new reviewer/maintainer group?

Reply:
I don't see aren't any groups in https://github.com/orgs/kubernetes-sigs/people (compare with https://github.com/orgs/kubernetes/people).